inference-engine/tests/ie_test_utils/functional_test_utils/layer_tests_summary/summarize.py

```diff
@@ -1,116 +1,122 @@
 # Copyright (C) 2018-2021 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-import xml.etree.ElementTree as ET
-from jinja2 import Environment, FileSystemLoader
 import argparse
 import os
+import logging
+import xml.etree.ElementTree as ET
+
+from jinja2 import Environment, FileSystemLoader
 from datetime import datetime
 
-parser = argparse.ArgumentParser()
-
-xml_help = """
-Paths to xml summary files from layer tests.
-In case of entries intersection, results will
-be merged basing on timestamp - entry from latest
-report is be kept.
-"""
-out_help = "Path where to save html report"
-
-parser.add_argument("--xml", help=xml_help, nargs="*", required=True)
-parser.add_argument("--out", help=out_help, default="")
-args = parser.parse_args()
-
-verified_operations = [
-    'Abs-0',
-    'Acos-0',
-    'Add-1',
-    'Asin-0',
-    'Assign-6',
-    'AvgPool-1',
-    'BatchNormInference-5',
-    'BinaryConvolution-1',
-    'Broadcast-1',
-    'Broadcast-3',
-    'Bucketize-3',
-    'CTCGreedyDecoder-0',
-    'CTCGreedyDecoderSeqLen-6',
-    'Concat-0',
-    'ConvertLike-1',
-    'Convolution-1',
-    'DeformableConvolution-1',
-    'DetectionOutput-0',
-    'Divide-1',
-    'ExperimentalDetectronDetectionOutput-6',
-    'ExperimentalDetectronGenerateProposalsSingleImage-6',
-    'ExperimentalDetectronPriorGridGenerator-6',
-    'ExperimentalDetectronROIFeatureExtractor-6',
-    'ExperimentalDetectronTopKROIs-6',
+logging.basicConfig()
+logger = logging.getLogger('Summarize')
+logger.setLevel(logging.INFO)
+
+
+def parse_arguments():
+    parser = argparse.ArgumentParser()
+
+    xml_help = """
+        Paths to xml summary files from layer tests.
+        In case of entries intersection, results will
+        be merged basing on timestamp - entry from latest
+        report is be kept.
+    """
+    out_help = "Path where to save html report"
+
+    parser.add_argument("--xml", help=xml_help, nargs="*", required=True)
+    parser.add_argument("--out", help=out_help, default="")
+
+    return parser.parse_args()
+
+
+def get_verified_op_list():
```

Comment:
It would be nice to move it to a separate file to not pollute source code and shorten lines number in one file.

Reply:
BTW, what was the reason to wrap this list to the function? In general, it will create a new list on each call instead of referring to the existing list if it will be just a variable.

Reply:
It will be good to create `utils` because it can become wheel in the future (The component is growing up)

Reply:
I doubt that wheel will be required, but anyway it may be stored not as `utils` but in e.g. `constants.py`.

Reply:
I fully agree with the approach. Please do it on your branch. I am afraid that if i separate it, it will be failed in CI